roles/pulp/handlers/main.yml

```diff
@@ -22,3 +22,4 @@
   notify: Restart pulpcore-api.service
   environment:
     PULP_SETTINGS: "{{ pulp_settings_file }}"
+    PULP_STATIC_ROOT: "{{ pulp_webserver_static_dir }}"
```

Comment:
Making sure we use the same `STATIC_ROOT` for all plugins

Reply:
So this is a setting that we only need during collectstatic (and for the webserver templates), and not at runtime? If needed at runtime, it should be in the pulp settings file.

Reply:
`STATIC_ROOT` already is on the settings file as it is set on pulpcore: https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L37
but dynaconf allow plugins to rewrite this: https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L229-L241
and the problem was: the installer is ready for working with the original `STATIC_ROOT`from pulpcore, which is `/var/lib/pulp/assets/`. When galaxy_ng overwrite `STATIC_ROOT` to `/app/test/` it broke due to lack of permissions of writing a dir on `/`

Reply:
so if we have pulpcore, pulp_ansible and galaxy_ng, each one defining one value for `STATIC_ROOT` it would be a problem, so this PR forces the `STATIC_ROOT` to be the one chosen on pulp role

Reply:
I see, but shouldn't we set it in the pulp_settings_file on disk that pulp_installer writes out? Like /etc/pulp/settings.py?

Reply:
good question! I don't have a strong opinion on that, @bmbouter @dkliban wdyt?

Reply:
Is whitenoise doing the serving of these assets?

Reply:
I don't know what you mean by whitenoise